lte/gateway/python/integ_tests/s1aptests/test_mobile_reachability_timer_with_mme_restart.py

```diff
@@ -0,0 +1,108 @@
+"""
+Copyright (c) 2016-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+"""
+
+import unittest
+import time
+
+import gpp_types
+import s1ap_types
+import s1ap_wrapper
+
+
+class TestMobileReachabilityTimerStatelessMme(unittest.TestCase):
+
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def test_mobile_reachability_timer_stateless_mme(self):
+        """
+        The test case validates Mobile Reachability Timer resumes the
+        configured timer value on MME restart
+        Step1 : UE attaches to network
+        Step2 : UE moves to Idle state
+        Step3 : Once MME restarts, MME shall resume
+                the Mobile reachability timer for remaining time, on expiry
+                MME starts the Implicit Detach Timer. On expiry of
+                Implicit Detach Timer, MME implicitly detaches UE.
+                MME shall delete the contexts locally
+        Step4 : Send Service Request, after Implicit Detach Timer Expiry
+                Expecting Service Reject, as MME has released the UE contexts
```

Comment:
Nit: Send Service Request, after Implicit Detach Timer expiry
                expecting Service Reject, as MME has released the UE context